src/Translate/Focusing.hs

```diff
@@ -65,6 +72,7 @@ isFocusedCmd eo (Apply loc _annot _kind prd cns) = Apply loc ApplyAnnotOrig (Jus
 isFocusedCmd _  (ExitSuccess loc)          = Just (ExitSuccess loc)
 isFocusedCmd _  (ExitFailure loc)          = Just (ExitFailure loc)
 isFocusedCmd _  (Jump loc fv)              = Just (Jump loc fv)
+isFocusedCmd eo (Method loc mn cn subst)   = Method loc mn cn <$> isFocusedSubst eo subst
 isFocusedCmd eo (Print loc prd cmd)        = Print loc <$> isValueTerm eo PrdRep prd <*> isFocusedCmd eo cmd
```

Comment:
We have to see whether we don't have to focus over the method call, i.e. `Show(2+2,k)` should be focused to ` 2 + 2 >> mu x. Show(x,k)`. But this is a separate PR.